site/src/pages/museum/about.astro

```diff
@@ -32,24 +32,55 @@ const headerNavLinks =
 
     <h2 class="masked hours"><span>Opening Hours</span></h2>
 
-    <div class="table">
-      <div class="row heading">
-        <div class="cell">Mon</div>
-        <div class="cell">Tue</div>
-        <div class="cell">Wed</div>
-        <div class="cell">Thu</div>
-        <div class="cell">Fri</div>
-        <div class="cell">Sat</div>
-        <div class="cell">Sun</div>
+    <div class="table__desktop" role="grid">
+      <div class="row" role="row">
+        <div class="cell" role="columnheader">Mon</div>
+        <div class="cell" role="columnheader">Tue</div>
+        <div class="cell" role="columnheader">Wed</div>
+        <div class="cell" role="columnheader">Thu</div>
+        <div class="cell" role="columnheader">Fri</div>
+        <div class="cell" role="columnheader">Sat</div>
+        <div class="cell" role="columnheader">Sun</div>
       </div>
-      <div class="row">
-        <div class="cell">9-5</div>
-        <div class="cell">9-5</div>
-        <div class="cell">9-5</div>
-        <div class="cell">9-5</div>
-        <div class="cell">9-5</div>
-        <div class="cell">8-8</div>
-        <div class="cell">9-8</div>
+      <div class="row" role="row">
+        <div class="cell" role="cell">9-5</div>
+        <div class="cell" role="cell">9-5</div>
+        <div class="cell" role="cell">9-5</div>
+        <div class="cell" role="cell">9-5</div>
+        <div class="cell" role="cell">9-5</div>
+        <div class="cell" role="cell">8-8</div>
+        <div class="cell" role="cell">9-8</div>
+      </div>
+    </div>
+
+    <div class="table__responsive" role="grid">
+      <div class="row" role="row">
+        <div class="cell" role="columnheader">Mon</div>
+        <div class="cell" role="cell">9-5</div>
+      </div>
+      <div class="row" role="row">
+        <div class="cell" role="columnheader">Tue</div>
+        <div class="cell" role="cell">9-5</div>
+      </div>
+      <div class="row" role="row">
+        <div class="cell" role="columnheader">Wed</div>
+        <div class="cell" role="cell">9-5</div>
+      </div>
+      <div class="row" role="row">
+        <div class="cell" role="columnheader">Fri</div>
+        <div class="cell" role="cell">9-5</div>
+      </div>
+      <div class="row" role="row">
+        <div class="cell" role="columnheader">Sat</div>
+        <div class="cell" role="cell">9-5</div>
+      </div>
+      <div class="row" role="row">
+        <div class="cell" role="columnheader">Sun</div>
+        <div class="cell" role="cell">9-5</div>
+      </div>
+      <div class="row" role="row">
+        <div class="cell" role="columnheader">Mon</div>
```

Comment:
Is this also intentional? You have Monday twice and no Thursday.

Reply:
Whoops. No, that was a mistake. Have fixed. Thanks.

And, yes, I was going to do something a bit more clever, but in the interest of time, a show/hide thing works.